docs/bc-registries-audit-notification-workflow.md

```diff
@@ -0,0 +1,82 @@
+# BC Registries Audit Notification Workflow
+
+The following workflow describes the steps to confirm and resolve a BC Registries Audit alert notification. Addition information regarding the associated error condition and its impact, along with details of the steps to resolve it can be found below.
+
+<p align="center">
+  <img src="https://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/bcgov/DITP-DevOps/main/docs/diagrams/bc-registries-audit-decision-tree.puml">
+</p>
+
+## What does an alert look like?
+
+![BC-Registries-Audit-Notification](./images/bc-registries-audit-notification1.png)
+![BC-Registries-Audit-Notification](./images/bc-registries-audit-notification.png)
+
+### What does it mean for the BC Reg and OrgBook data to match (pass the audit)?
+
+There should be no discrepancy of data between BC Reg and Orgbook data inorder to pass the audit. That means corporation name, corporation status, corporation type,business number,registration number,jurisdiction number should be matching between BC Reg and Orgbook exactly to pass the audit.
+
+
+### What is the impact of the data being out of sync?
+
+When the data is out of sync the dispalyed data of companies in orgbook is incorrect.In order To synchronize their information we want to make sure that we're providing them with  the correct information.If its incorrect, we're misrepresenting the actual corporate data and that needs to be fixed.
+
+
+### What affects the synchronization of the data?
+
+Synchronization of the data is affected by updates being made in BC registries. Some of those updates are not associated with database event records. So then if there's no database event record, then the BC registries agent doesn't know to pick up those changes and then send them over to the org book. So that's that's one major discrepancy is when things are done manually on the VC registry side.There is no way for us to detect that that's actually happened until the audit is done.
+
+Other things that affects synchronization is timing. So in case where the audit says that there's a missing record in BC in Org book, and then you go in later to take a look at that data and it's there, that's really a timing issue. So basically what happened is the audit captured the fact that data was in BC Reg but not in Org book before the data was actually sent over to the orgbook.So the timing issue is between when the audit script is run and when data is actually being synchronized.
+
+There are bunch of services always running. So if those services are interrupted or Internet connectivity is down, that can affect the synchorization of data for a little while.
+
+## Digital Trust Monitoring Services OCP Environments
+
+Links to the Deployment Configurations Console can be found here; [Administrator - Deployment Configurations Console](./digital-trust-monitoring-services-environments)
+
+## BC Registries Agent Configurations `./manage` script
```

Comment:
Missing a corresponding section for the orgbook-configurations `./manage` script and the link to the `DeleteTopic` command.